src/input/input.js

```diff
@@ -131,5 +131,6 @@ export function hiddenTextarea() {
   // If border: 0; -- iOS fails to open keyboard (issue #1287)
   if (ios) te.style.border = "1px solid black"
   disableBrowserMagic(te)
+  te.setAttribute("aria-hidden","true")
```

Comment:
Hiding the input element from the screen reader seems a _very_ bad idea. What is the purpose of this line?

Reply:
Sorry, I probably misunderstood what is this textarea for. Maybe it would be good to just add aria-label for this textarea?
If so, what should be in the aria-label?

Reply:
This creates the main focused element when `inputMode` is `"textarea"`. Its label can be set with [this option](https://codemirror.net/doc/manual.html#option_screenReaderLabel).

Reply:
Ok, I reverted this change and set the label locally. That's all for this pull request